`--- /home/app/data/titles/preprocessed/xml/02/2021/04/2021-04-28_8bf0ce26.xml	2021-12-20 17:27:56.586890248 -0500
+++ tmp/title_version_2_2021-04-28_8bf0ce26_preprocessed.xml	2021-12-20 17:54:24.368804405 -0500
@@ -18021,13 +18021,13 @@
 <HEAD> CHAPTER XI—DEPARTMENT OF DEFENSE</HEAD>

 <DIV4 N="A" TYPE="SUBCHAP">
-<HEAD>SUBCHAPTER A—GENERAL MATTERS AND DEFINITIONS
+<HEAD>SUBCHAPTER A—GENERAL MATTERS AND DEFINITIONS</HEAD>

 <SOURCE>
 <HED>Source:</HED><PSPACE>85 FR 51160, Aug. 19, 2020, unless otherwise noted.


-</PSPACE></SOURCE></HEAD>
+</PSPACE></SOURCE>

 <DIV5 N="1100-1103" TYPE="PART">
 <HEAD>PARTS 1100-1103 [RESERVED]
@@ -19273,7 +19273,7 @@


 <DIV4 N="C" TYPE="SUBCHAP">
-<HEAD>Subchapter C—AWARD FORMAT AND NATIONAL POLICY TERMS AND CONDITIONS FOR ALL GRANTS AND COOPERATIVE AGREEMENTS
+<HEAD>Subchapter C—AWARD FORMAT AND NATIONAL POLICY TERMS AND CONDITIONS FOR ALL GRANTS AND COOPERATIVE AGREEMENTS</HEAD>

 <SOURCE>
 <HED>Source:</HED><PSPACE>85 FR 51160, Aug. 19, 2020, unless otherwise noted.
@@ -19281,7 +19281,7 @@



-</PSPACE></SOURCE></HEAD>
+</PSPACE></SOURCE>

 <DIV5 N="1120" TYPE="PART">
 <HEAD>PART 1120—AWARD FORMAT FOR DOD GRANTS AND COOPERATIVE AGREEMENTS
@@ -20349,7 +20349,7 @@


 <DIV4 N="D" TYPE="SUBCHAP">
-<HEAD>SUBCHAPTER D—ADMINISTRATIVE REQUIREMENTS TERMS AND CONDITIONS FOR COST-TYPE GRANTS AND COOPERATIVE AGREEMENTS TO NONPROFIT AND GOVERNMENTAL ENTITIES
+<HEAD>SUBCHAPTER D—ADMINISTRATIVE REQUIREMENTS TERMS AND CONDITIONS FOR COST-TYPE GRANTS AND COOPERATIVE AGREEMENTS TO NONPROFIT AND GOVERNMENTAL ENTITIES</HEAD>



@@ -20365,7 +20365,7 @@



-</PSPACE></SOURCE></HEAD>
+</PSPACE></SOURCE>

 <DIV5 N="1126" TYPE="PART">
 <HEAD>PART 1126—SUBCHAPTER D OVERVIEW
